src/pages/PeoplePage/PersonListPage/PersonListPage.tsx

```diff
@@ -1,27 +1,21 @@
 import { IonContent, IonHeader, IonPage } from '@ionic/react'
 import { ReactElement, useState } from 'react'
-import { useQuery } from 'react-query'
 
 import { PeopleMenuBar } from '../../../components/PeopleMenuBar'
-import { getProcessDefinitions } from '../../../lib/queries/getProcessDefinitions'
+import { PersonList } from '../../../components/PersonList'
+import { useAuth } from '../../../lib/useAuth'
 
 export function PersonListPage(): ReactElement {
+  const { user } = useAuth()
   const [search, setSearch] = useState('')
 
-  const { data: processDefinition } = useQuery(
-    ['definitions', { type: 'process' }],
-    getProcessDefinitions
-  )
-
   return (
     <IonPage>
       <IonHeader>
-        <PeopleMenuBar onChange={(value): void => setSearch(value)} />
+        <PeopleMenuBar onChange={setSearch} />
       </IonHeader>
       <IonContent fullscreen>
-        {search}
-        {processDefinition != null &&
-          Object.keys(processDefinition).map((key) => <div>{key}</div>)}
+        {user != null && <PersonList search={search} user={user} />}
```

Comment:
this page is never rendered where user is null.

Reply:
The page will be there but the list just won't show if there's no user. What should it display if there are no processes assigned to a user?